README.rst

```diff
@@ -105,6 +105,8 @@ They can be used when installing in order to include additional dependencies, e.
 
     $ pip install jsonschema'[format]'
 
+You will also want to read the `format validation documentation <https://python-jsonschema.readthedocs.io/en/latest/validate/#validating-formats>`_.
```

Comment:
This is too ominous IMHO. Also I think a relative link with a ref is better? I just accidentally clicked myself into the official docs and was surprised why it's old.

You'd add something like `.. _validating-formats:` _right before the heading_ and then you can use `:ref:BACKTICKvalidating-formatsBACKTICK` or something similar.

```suggestion
Be aware that the mere presence of these dependencies – or even the specification of ``format`` checks in a schema – do **not** activate format checks (as per specification).
Please read the `format validation documentation <https://python-jsonschema.readthedocs.io/en/latest/validate/#validating-formats>`_ for details.
```

Reply:
This is the README, so the relative links don't work (they'll work in sphinx from RTD but not when clicking in the README itself, which doesn't descend into the doc folder). This is why I want to kill the readme entirely personally (and just link to the docs), rather than extend it :/

(Other change looks good though thanks).